src/ucp/api/ucp.h

```diff
@@ -2616,6 +2705,46 @@ ucs_status_t ucp_tag_recv_nbr(ucp_worker_h worker, void *buffer, size_t count,
                               ucp_tag_t tag_mask, void *req);
 
 
+/**
+ * @ingroup UCP_COMM
+ * @brief Non-blocking tagged-receive operation.
+ *
+ * This routine receives a messages that is described by the local address @a
+ * buffer, size @a count, and @a datatype object on the @a worker. The tag
```

Comment:
probably worth mention that these are `info` fields

Reply:
ok